src/algorithmForNN/README.md

```diff
@@ -37,9 +33,19 @@ Possible parameters:
                         The minimum area of a plane to segment it.
 ```
 
-## **What's interesting inside?**
+## **Usage example**
+
+```bash
+$ python main.py --path_to_data_folder /home/pavel/dataset/sequences/00/velodyne \ 
+  --path_to_label_folder /home/pavel/dataset/sequences/00/labels \
+  --path_to_new_label_folder home/pavel/result \
```

Comment:
I see two label folders, but the first definition of the tool only says that we generate semi-ready labels, it is better to explain why we need two folders.